### PR TITLE
feat: adds missing translation keys

### PR DESCRIPTION
**Pull request by pedrolamas**
Signed-off-by: Pedro Lamas <pedrolamas@gmail.com>

### CODE REVIEW

File: src/components/ui/AppCardDrag.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <v-card-title
       class="card-title card-heading py-1 draggable"
     >
-      Drag Me
+      {{ $t('app.general.label.drag_me') }}
```
Comment (matmen) [inline src/components/ui/AppCardDrag.vue:6]: I don't see this file being imported anywhere - maybe we should just remove it?

File: src/components/widgets/filesystem/FileSystemContextMenu.vue [hunk 2]
```diff
@@ -27,9 +27,7 @@
               <v-list-item-icon>
                 <v-icon>$printer</v-icon>
               </v-list-item-icon>
-              <v-list-item-title>
-                Print
-              </v-list-item-title>
+              <v-list-item-title> {{ $t('app.general.btn.print') }}</v-list-item-title>
```
Comment (matmen) [inline src/components/widgets/filesystem/FileSystemContextMenu.vue:30]:
```suggestion
              <v-list-item-title>{{ $t('app.general.btn.print') }}</v-list-item-title>
```

**Review by matmen — CHANGES_REQUESTED** [review]

File: src/components/ui/AppCardDrag.vue (hunk 1 above)
Comment (pedrolamas) [reply src/components/ui/AppCardDrag.vue:6]: Very good point there, I was actually surprised as I didn't remember seeing "drag me" string before... I'll remove the file and translation!

Comment (pedrolamas) [reply src/components/ui/AppCardDrag.vue]: Done!

**Review by matmen — APPROVED** [review]

**Pull request merged**